src/main/java/jenkins/plugins/gerrit/workflow/GerritCommentStep.java

```diff
@@ -73,16 +73,22 @@ protected Execution(@Nonnull StepContext context) throws IOException, Interrupte
 
     @Override
     protected Void run() throws Exception {
+      String gerritApiUrl = envVars.get("GERRIT_API_URL");
+      String credentialsId = envVars.get("GERRIT_CREDENTIALS_ID");
       String branch = envVars.get("BRANCH_NAME");
+
+      if (gerritApiUrl == null) {
+        echo("GERRIT_API_URL is not available, disabling Gerrit integration");
```

Comment:
How did you manage to come up with this situation?
The Gerrit API URL should always be extracted from the SCM URL ... and if it is not, it means we have a bug :-(

However, better a message that 'GERRIT_API_URL is not available' than a NPE, agreed :-)

Reply:
As I wrote to you in jenkins, I am interested to use this plugin also on my own projects (non-multibranch) as there is lack of functionality of triggers, so I was happy that you used environment variables which can be set by any project.